sigridci/sigridci/platform.py

```diff
@@ -0,0 +1,33 @@
+# Copyright Software Improvement Group
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+
+class Platform:
+    @staticmethod
+    def isGitHub():
+        return "GITHUB_REPOSITORY" in os.environ
+
+    @staticmethod
+    def isGitLab():
+        return "CI_PROJECT_PATH" in os.environ
```

Comment:
We could use variable `GITLAB_CI` instead. The name `CI_PROJECT_PATH` is rather generic, there might be other environments that also use it. On GitLab, `GITLAB_CI` is defined and typically has the value `true`.

Reply:
Fair point, changed. By that logic the Azure one is maybe also a bit generic, but from briefly looking at their list I have the feeling they're *all* generic like that.